docs/api/cypress-api/ensure.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Cypress.ensure
+---
+
+`Cypress.ensure.method()` is a collection of helper methods for verifying
+conditions while executing Cypress commands. They are mostly useful when writing
+[custom queries](/api/cypress-api/custom-queries) or
+[commands](/api/cypress-api/custom-commands).
+
+Most functions on `Cypress.ensure` accept `subject` argument, and assert
+something about it, throwing an error if it doesn't match. They have no return
```

Comment:
```suggestion
something about it; if the assertion fails, an error is thrown. They have no return
```

Reply:
Applied Zach's suggestion to redo these lines.